sdk/execution/metrics.go

```diff
@@ -0,0 +1,56 @@
+package executionsdk
+
+import (
+	"github.com/go-kit/kit/metrics"
+	prometheus "github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+var m *metric
```

Comment:
How do this object is shared to tendermint's prometheus? Is it working with public singleton?

Reply:
just register globally when initializing it with `prometheus.NewCounterFrom`